relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -492,12 +505,179 @@ impl VisitorMut for NormalizeVisitor {
+                if let Some(db_name) = db_name {
+                    Self::scrub_name(db_name);
+                }
+                Self::scrub_statement_filter(filter);
+            }
+            Statement::ShowCollation { filter } => Self::scrub_statement_filter(filter),
+            Statement::Use { db_name } => Self::scrub_name(db_name),
+            Statement::StartTransaction { .. } => {}
+            Statement::SetTransaction { .. } => {}
+            Statement::Comment { comment, .. } => *comment = None,
+            Statement::Commit { .. } => {}
+            Statement::Rollback { savepoint, .. } => {
+                if let Some(savepoint) = savepoint {
+                    Self::erase_name(savepoint);
+                }
+            }
+            Statement::CreateSchema { .. } => {}
+            Statement::CreateDatabase {
+                location,
+                managed_location,
+                ..
+            } => {
+                *location = None;
+                *managed_location = None;
+            }
+            Statement::CreateFunction { .. } => {}
+            Statement::CreateProcedure { .. } => {}
+            Statement::CreateMacro { .. } => {}
+            Statement::CreateStage { comment, .. } => *comment = None,
+            Statement::Assert { .. } => {}
+            Statement::Grant {
+                grantees,
+                granted_by,
+                ..
+            } => {
+                Self::simplify_compound_identifier(grantees);
+                *granted_by = None;
+            }
+            Statement::Revoke {
+                grantees,
+                granted_by,
+                ..
+            } => {
+                Self::simplify_compound_identifier(grantees);
+                *granted_by = None;
+            }
+            Statement::Deallocate { name, .. } => {
+                Self::scrub_name(name);
+            }
+            Statement::Execute { name, .. } => Self::scrub_name(name),
+            Statement::Prepare { name, .. } => Self::scrub_name(name),
+            Statement::Kill { id, .. } => *id = 0,
+            Statement::ExplainTable { .. } => {}
+            Statement::Explain { .. } => {}
+            Statement::Merge { .. } => {}
+            Statement::Cache { .. } => {}
+            Statement::UNCache { .. } => {}
+            Statement::CreateSequence { .. } => {}
+            Statement::CreateType { .. } => {}
+            Statement::Pragma { .. } => {}
+            Statement::LockTables { tables } => {
+                for table in tables {
+                    let LockTable {
+                        table,
+                        alias,
+                        lock_type: _,
+                    } = table;
+                    Self::scrub_name(table);
+                    if let Some(alias) = alias {
+                        Self::scrub_name(alias);
+                    }
+                }
+            }
+            Statement::UnlockTables => {}
```

Comment:
These have not caused high cardinality in the past, but it's better to list them explicitly in case a new variant is introduced.